pkgs/top-level/all-packages.nix

```diff
@@ -9743,14 +9743,17 @@ in
 
   bam = callPackage ../development/tools/build-managers/bam {};
 
-  bazel = callPackage ../development/tools/build-managers/bazel {
+  bazel_base = recurseIntoAttrs (callPackages ../development/tools/build-managers/bazel {
```

Comment:
I think the convention (or at least common practice) is to call this `bazelPackages`.

Reply:
Resolved